src/containers/Home/HomeHero/hello-world.ts

```diff
@@ -39,6 +39,7 @@ const HelloWorld: {[isoLangCode: string]: string} = {
   ne: 'नमस्कार संसार',
   nl: 'Hallo Wereld',
   no: 'Hallo Verden',
+  od: 'ନମସ୍କାର ବିଶ୍ୱବାସି'
```

Comment:
There is a comma missing and also please check the ISO code again.

Reply:
I have added the comma as well as corrected the ISO code